CHANGES.rst

```diff
@@ -339,6 +339,17 @@ astropy.tests
 astropy.time
 ^^^^^^^^^^^^
 
+- Ensure multiplication of a ``TimeDelta`` instance with a ``Quantity`` with
```

Comment:
You've done a *much more sweeping* improvement than just the case of time squared.  More like "Fixed a number of issues to ensure a consistent output type resulting from multiplication or division involving a ``TimeDelta`` instance.  The output is now always a ``TimeDelta`` if the result is a time unit (like ``u.s`` or ``u.d``), otherwise it will be a ``Quantity``."

(Have I characterized that in a technically correct way?)

Reply:
Both of these should also get copied into API changes as well since they do change the output or exception type.

Reply:
OK, done